### ISSUE TITLE
Struggling with SDXL LoRA training

### ISSUE THREAD

**Issue by MarianoMolina** [bmaltais/kohya_ss#1987]
Hey, 
Not sure if this is a good place for this, but honestly a bit baffled at this point since I've been trying for a few weeks now so I'm sure there's something I'm missing. 

I'm training a LoRA of a man on SDXL: 

![imagen](https://github.com/bmaltais/kohya_ss/assets/15909746/4608f657-6bde-4200-bd0c-5a44c669e0f7)

Here's my presets for the parameters:

{
    "LoRA_type": "Standard",
    "LyCORIS_preset": "full",
    "adaptive_noise_scale": 0,
    "additional_parameters": "",
    "block_alphas": "",
    "block_dims": "",
    "block_lr_zero_threshold": "",
    "bucket_no_upscale": false,
    "bucket_reso_steps": 64,
    "cache_latents": true,
    "cache_latents_to_disk": true,
    "caption_dropout_every_n_epochs": 0.0,
    "caption_dropout_rate": 0.05,
    "caption_extension": ".txt",
    "clip_skip": "1",
    "color_aug": false,
    "constrain": 0.0,
    "conv_alpha": 1,
    "conv_block_alphas": "",
    "conv_block_dims": "",
    "conv_dim": 1,
    "debiased_estimation_loss": false,
    "decompose_both": false,
    "dim_from_weights": false,
    "down_lr_weight": "",
    "enable_bucket": true,
    "epoch": 20,
    "factor": -1,
    "flip_aug": false,
    "full_bf16": false,
    "full_fp16": false,
    "gradient_accumulation_steps": "1",
    "gradient_checkpointing": true,
    "keep_tokens": "0",
    "learning_rate": 3e-05,
    "logging_dir": "[removed]",
    "lora_network_weights": "",
    "lr_scheduler": "constant",
    "lr_scheduler_args": "",
    "lr_scheduler_num_cycles": "",
    "lr_scheduler_power": "",
    "lr_warmup": 0,
    "max_bucket_reso": 2048,
    "max_data_loader_n_workers": "0",
    "max_grad_norm": 1,
    "max_resolution": "1024,1024",
    "max_timestep": 1000,
    "max_token_length": "75",
    "max_train_epochs": "20",
    "max_train_steps": "",
    "mem_eff_attn": false,
    "mid_lr_weight": "",
    "min_bucket_reso": 256,
    "min_snr_gamma": 5,
    "min_timestep": 0,
    "mixed_precision": "bf16",
    "model_list": "stabilityai/stable-diffusion-xl-base-1.0",
    "module_dropout": 0,
    "multires_noise_discount": 0,
    "multires_noise_iterations": 0,
    "network_alpha": 64,
    "network_dim": 64,
    "network_dropout": 0,
    "no_token_padding": false,
    "noise_offset": 0,
    "noise_offset_type": "Original",
    "num_cpu_threads_per_process": 2,
    "optimizer": "Adafactor",
    "optimizer_args": "scale_parameter=False relative_step=False warmup_init=False",
    "output_dir": "[removed]",
    "output_name": "[removed]",
    "persistent_data_loader_workers": false,
    "pretrained_model_name_or_path": "stabilityai/stable-diffusion-xl-base-1.0",
    "prior_loss_weight": 1.0,
    "random_crop": false,
    "rank_dropout": 0,
    "rank_dropout_scale": false,
    "reg_data_dir": "[removed]",
    "rescaled": false,
    "resume": "",
    "sample_every_n_epochs": 1,
    "sample_every_n_steps": 0,
    "sample_prompts": "# prompt 1\nmasterpiece, best quality, [keyword], man, handsome, professional photography --n low res, bad quality, ugly, deformed --d 1\n# prompt 2\nmasterpiece, best quality, man, handsome, professional photography --n low res, bad quality, ugly, deformed --d 1",
    "sample_sampler": "euler_a",
    "save_every_n_epochs": 1,
    "save_every_n_steps": 0,
    "save_last_n_steps": 0,
    "save_last_n_steps_state": 0,
    "save_model_as": "safetensors",
    "save_precision": "bf16",
    "save_state": false,
    "scale_v_pred_loss_like_noise_pred": false,
    "scale_weight_norms": 0,
    "sdxl": true,
    "sdxl_cache_text_encoder_outputs": false,
    "sdxl_no_half_vae": true,
    "seed": "",
    "shuffle_caption": false,
    "stop_text_encoder_training_pct": 0,
    "text_encoder_lr": 3e-05,
    "train_batch_size": 1,
    "train_data_dir": "",
    "train_norm": false,
    "train_on_input": true,
    "training_comment": "20 epochs, 3 batch size",
    "unet_lr": 3e-05,
    "unit": 1,
    "up_lr_weight": "",
    "use_cp": false,
    "use_scalar": false,
    "use_tucker": false,
    "use_wandb": false,
    "v2": false,
    "v_parameterization": false,
    "v_pred_like_loss": 0,
    "vae": "", > EMPTY
    "vae_batch_size": 0,
    "wandb_api_key": "",
    "weighted_captions": false,
    "xformers": "xformers"
  }
(Deleted folder references, but they are set up ofc). 

The problem I'm having is that the prompt 2 (masterpiece, best quality, man, handsome, professional photography --n low res, bad quality, ugly, deformed --d 1) sample, which should not activate the LoRa since it lacks the keword, I'm getting weird generations like this:

![mariano_v9_sdxl_20240219184330_e000002_01_1](https://github.com/bmaltais/kohya_ss/assets/15909746/d62e9384-f73c-4387-a524-131b7ddfdce3)

That's from the first epoch. 

Feels like its missing the VAE or something, given how cooked the images look. Any idea what I am doing wrong?
